test/rss.cpp

```diff
@@ -249,4 +249,23 @@ TEST_CASE("set_rssurl checks if query feed has a valid query", "[rss]") {
 	}
 }
 
+TEST_CASE("rss_item::sort_flags() cleans up flags", "[flag]") {
+	configcontainer cfg;
+	cache rsscache(":memory:", &cfg);
+	std::shared_ptr<rss_item> item = std::make_shared<rss_item>(&rsscache);
```

Comment:
I just noticed: you don't really need a pointer here as you don't pass it to `rss_feed`'s methods.